src/scripts/main.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
-}
+  return new Promise(resolve => {
+    element.addEventListener(eventName, () => {
+      resolve(`It was ${eventName} on the element:
+        ${element.nodeName}, id: ${element.id}.`);
+    });
+  });
+};
 
 const printMessage = (message) => {
-// write your code here
+  const div = document.createElement('div');
+
+  div.className('message');
```

Comment:
```suggestion
  div.className = 'message';
```
or some other method to set class. The current one is incorrect and your solution doesn't work, it throws Uncaught TypeError instead